actions/consul/helm_deploy_consul/action.yaml

```
@@ -0,0 +1,59 @@
+name: "Helm deploy Action for Consul"
+description: "GitHub Action for install/upgrade Consul using Helm"
+inputs:
+  deploy_mode:
+    description: 'Install or update'
+    required: true
+    default: 'install'
+  restricted:
+    description: 'true or false'
```

The parameter is `boolean`, so it is clear it should be either `true` or `false`.

The description should better explain the meaning of the parameter.

For instance, you could reference `helm_deploy.restricted` or copy the description as follows
```suggestion
    description: |
      - 'true': helm installation by user with restricted rights
      - 'false': helm installation by cluster admin user
```

---

added references